### PR TITLE
Added twitter icon

### LINKED ISSUE #11: Replace gitter icon with twitter icon
**Describe the bug/feature**

Currently, we show gitter icon in the navigation bar/footer of the website, which is serving no purpose since, development of the website is of no use to a common rotaractor. So it would be better to replace the gitter icon with twitter icon.

**To Reproduce(for bug)**

**Screenshots(for bug)**

As can be seen the gitter icon in the rightmost end of the navigation bar. That should be replaced with twitter icon.
Screenshot from 2019-08-15 22-26-36
Screenshot from 2019-08-15 22-28-14
Screenshot from 2019-08-15 22-27-55

**Device Details(for bug)**

**Other Comments**

@asharani97 Can you please fix this issue?

### PR DESCRIPTION
**Pull request by asharani97**
#### Type of Change:

- [x] Code
- [ ] Blog
- [ ] Resources

#### Description

Fixes #11

### CODE REVIEW

File: _includes/foot.html [hunk 1]
```diff
@@ -44,9 +45,9 @@ <h3>Social</h3>
                   <div class="logo-right-arm-f"></div>
               </div>
             </div>
-          </div>
-            <span class="label">gitter</span></a>
-          </li>
+          </div>-->
```
Comment (czgdp1807) [inline _includes/foot.html]: Please delete L38 - L48 rather than commenting them out. They are of no use after this PR.

File: _includes/nav.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
           <div class="logo-right-arm"></div>
       </div>
     </div>
-  </div>
-    <span class="label">gitter</span></a>
+  </div>-->
```
Comment (czgdp1807) [inline _includes/nav.html]: Delete L11 - L20.

**Comment by czgdp1807** [comment]
Thanks for the PR.
Changes to be made:

1. Delete https://github.com/rtrclubiitj/rtrclubiitj.github.io/blob/master/assets/css/main.scss#L73-L483 code. It's of no use after removal of the gitter icon.

2. Replace `Fixes #ISSUE_NUMBER_HERE` with `Fixes #11` in [the description](https://github.com/rtrclubiitj/rtrclubiitj.github.io/pull/13#issue-308239433).

3. Update the heading to a more descriptive line. `Merge twitter update` doesn't making much sense for this change.

4. Add `twitter: club_iit` [here](https://github.com/rtrclubiitj/rtrclubiitj.github.io/blob/master/_config.yml#L18).

Some suggestions for future PRs

1. Never work from `master`. Always checkout to a new branch for making changes to the code.

2. Wherever, possible write short but descriptive commit messages like, `added twitter icon`.

That's all.

**Comment by czgdp1807** [comment]
Please execute `git pull origin master` on `master` branch provided the following happens,

```bash
git remote get-url origin
https://github.com/rtrclubiitj/rtrclubiitj.github.io/
```

File: _includes/foot.html (hunk 1 above)
Comment (asharani97) [reply _includes/foot.html]: okay

**Comment by czgdp1807** [comment]
Why closed?

**Comment by czgdp1807** [comment]
It's ready to go.

**Comment by czgdp1807** [comment]
Please update your `master` branch by pulling the changes from master of the main repository.

**Comment by asharani97** [comment]
okay

**Pull request merged**